src/Education/Commands/RuleDescriptionUserControl.xaml

```diff
@@ -1,10 +1,123 @@
 ﻿<UserControl x:Class="SonarLint.VisualStudio.Education.Commands.RuleDescriptionUserControl"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
-             xmlns:vsImaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
-             xmlns:vsCatalog="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.ImageCatalog"
-             xmlns:vsTheming="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Imaging"
-             xmlns:vsUtilities="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Utilities"
-             xmlns:core="clr-namespace:SonarLint.VisualStudio.Core.WPF;assembly=SonarLint.VisualStudio.Core">
+             xmlns:vsshell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
+             xmlns:vsTheming="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+             >
+
+    <UserControl.Resources>
+        <!-- Defines the standard set of style used by the document elements -->
+        
+        <!--<Style TargetType="Paragraph">
+            <Setter Property="Margin" Value="3" />
+        </Style>-->
+        <Style TargetType="FlowDocument">
```

Comment:
Adds an initial set of style properties. They need some fine-tuning, but the documents are at least readable in both light and dark modes.

![image](https://user-images.githubusercontent.com/10757559/216385000-316e1883-f101-4ae3-9531-d8f6781b13d6.png)


![image](https://user-images.githubusercontent.com/10757559/216385223-ac608e60-7a03-487b-bada-54ba02b218bd.png)

Reply:
Very nice!!